README.md

```diff
@@ -3,7 +3,11 @@
 Service and clients to split audiences into buckets based on an arbitrary "user_id".
 
  * Source code for the service is all under `./service`. Run `make build` to create the executable `./service/bin/bucket_server`. This takes one parameter: the path to a config file.
- * Source code for the Android client can be found in `./clients/android`.  Execute `./gradlew clean && ./gradlew install && ./gradlew bintrayUpload` to upload to jcenter.
+ * Source code for the Android client can be found in `./clients/android`. to use android client SDK just include the following in your gradle file 
+ ```sh
+compile 'de.rocketinternet:android.bucket:0.2'
+``` 
+more details android instructions can be found at the bottom of the page.
```

Comment:
Does this parse correctly? As in, is "more details..." part of the bullet point above or something else?